sign.css

```diff
@@ -0,0 +1,55 @@
+
+body {font-family: Arial, Helvetica, sans-serif; width: 50%; margin-left: 25%; margin-right: 25%; }
+form {border: 3px solid #f1f1f1; background-color: #94ffff  ;}
+
+input[type=text], input[type=password] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+}
+
+button {
+  background-color: #008080;
```

Comment:
Use the variable itself not it's Hexa representation `var(--navbar-bg-color)`

Reply:
done